multivac/fetch.py

```diff
@@ -25,13 +25,7 @@
     raise ValueError('repo_path must be in the form owner/repository')
 args.owner, args.repo = args.repo_path.split('/', 1)
 
-token_file = 'token.txt'
-if not os.path.exists(token_file):
-    raise RuntimeError('{file} is not exists'.format(file=token_file))
-if not os.path.isfile(token_file):
-    raise RuntimeError('{file} is not a regular file'.format(file=token_file))
-with open(token_file, 'r') as f:
-    token = f.read().strip()
+token = os.environ['MULTIVAC_GITHUB_TOKEN']
```

Comment:
Add an `assert token, 'error message'` please
If the token is `None`, the user will immediately see the error.

Reply:
Fixed